mappings/project_institution.yaml

```
@@ -18,6 +18,7 @@ Michigan: "University of Michigan"
 Mines: "Colorado School of Mines"
 Mizzou: "University of Missouri"
 MSU: "Montana State University"
+MTU: "Minnesota Technological University"
```

```suggestion
MTU: "Michigan Technological University"
```

The merged project says Michigan